### PR TITLE
Arch User Repository PKGBUILD

### PR DESCRIPTION
**Pull request by christophfink**
This *should* now create a PKGBUILD (build recipe) to be uploaded to the Archlinux User Repository. Not quite sure still how to debug Github Actions.

### CODE REVIEW

**Comment by christophfink** [comment]
This is a bit of a troublesome PR, as for testing it, we might have to create an actual release. Maybe (maybe!), we can also trigger it manually on an existing version tag (:crossed_fingers: )

File: .github/workflows/PKGBUILD.template [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+pkgname=python-r5py
+_name=${pkgname#python-}
+pkgdesc="Rapid Realistic Routing with R5 in Python"
+url="https://github.com/r5py/r5py"
+
+pkgver=
+pkgrel=1
+
+arch=("any")
+license=("GPL")
```
Comment (HTenkanen) [inline .github/workflows/PKGBUILD.template:12]: Is it possible to have the dual license here as well?

**Review by HTenkanen — APPROVED** [review]
I don't really have the required expertice to properly evaluate the linux build, but looks good as far as I can understand.

File: .github/workflows/PKGBUILD.template (hunk 1 above)
Comment (christophfink) [reply .github/workflows/PKGBUILD.template]: I wasn’t sure, and then I forgot to look it up, but yes!

**Comment by christophfink** [comment]
Screenshot from 2022-05-19 20-11-40

So, we’ll need a version tag to debug this. I’ll triple read over this still, and also add a debian and rpm package script, first

**Pull request merged**